deploy/helm/values.yaml

```diff
@@ -185,12 +185,7 @@ otel:
 
     # This filter is applied after metric processing, it is the place where metrics could be filtered out
     # https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/filterprocessor for configuration reference
-    filter:
-      exclude:
-        match_type: regexp
-        metric_names:
-          - .*_temp
-          - apiserver_request_total
```

Comment:
Let's just hard-code this. There is no scenario where we want customers to change these default filters. That way we don't have to deal with default filters in old vs. new filtering syntax.